specificatie/zakelijk-gerechtigden.yaml

```diff
@@ -151,6 +162,9 @@ components:
     ZakelijkGerechtigdeHalCollectieEmbedded:
       type: "object"
       properties:
+        zakelijkGerechtigdenUitSplitsing:
```

Comment:
hernoemen naar indicatieZakelijkGerechtigdenUitSplitsing. Dan is het conform de benaming bij de (meeste) boolean properties

Reply:
Aangepast